packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerHeader.tsx

```diff
@@ -34,11 +37,11 @@ const styles = {
     container: css`
         position: relative;
     `,
-    header: css`
+    header: (headerSettings: HeaderSettings | undefined) => css`
```

Comment:
Can we mark headerSettings as optional using a `?` instead?

Reply:
Same as above on this one